pyproject.toml

```diff
@@ -1,37 +1,37 @@
 [build-system]
 requires = [
-    "hatchling",
-    "jupyterlab~=3.1",
+    "hatchling>=1.21.1",
+    "jupyterlab>=4.0.0,<5",
+    "hatch-nodejs-version>=0.3.2",
```

Comment:
migration script missed`hatch-nodejs-version`, but the cookiecutter has it, so added it.